Adjusting info on chilling (#3829)
w3f/polkadot-wiki: docs/maintain/maintain-guides-how-to-chill.md

slug: ../maintain-guides-how-to-chill
---

> Staking bonds can be in any one of the three states: validating, nominating, or chilled (neither validating or nominating).  When a staker wants to temporarily pauses their active engagement in staking but does not want to unbond their funds, they can choose to "chill" their involvement and keep their funds bonded.

Reviewer: ```suggestion
Staking bonds can be in any of the three states: validating, nominating, or chilled (neither validating nor nominating).  When a staker wants to temporarily pause their active engagement in staking but does not want to unbond their funds, they can choose to "chill" their involvement and keep their funds bonded.
```

:::

![staking](../assets/NPoS/staking-keys_stash_controller.png)
## Consideration for Staking Election

> A bond that is actively participating in staking but chilled would continue to participate in staking for the rest of the current era.  If the bond was chilled in sessions 1 through 4 and continues to be chilled for the rest of the era it would NOT be selected for election in the next era.  If a bond was chilled for the entire of session 5 it would not be considered in the next election.  If the bond was chilled in session 6 its participation in the next era's election would be dependent on its state in session 5.

Reviewer: ```suggestion
A bond that is actively participating in staking but chilled would continue to participate in staking for the rest of the current era.  If the bond was chilled in sessions 1 through 4 and continues to be chilled for the rest of the era, it would NOT be selected for election in the next era.  If a bond was chilled for the entire session 5, it would not be considered in the next election.  If the bond was chilled in session 6, its participation in the next era's election would depend on its state in session 5.
```

## Chilling as a Nominator

When you chill after being a nominator, your nominations will be reset. This means that when you decide to start nominating again you will need to select validators to nominate once again. These can be the same validators if you prefer, or, a completely new set. Just be aware - your nominations will not persist across chills.

Your nominator will remain bonded when it is chilled. When you are ready to nominate again, you will not need to go through the whole process of bonding again, rather, you will issue a new nominate call that specifies the new validators to nominate.

## Chilling as a Validator

> When you voluntarily chill after being a validator, your nominators will remain.  As long as your nominators make no action, you will still have the nominations when you choose to become an active validator once again.  You bond however would not be listed as a nominable validator thus any nominators issuing new or revisions to existing nominations would not be able to select your bond.

Reviewer: By bond, do you mean the validator node?

```suggestion
When you voluntarily chill after being a validator, your nominators will remain.  If your nominators take no action, you will still have their nominations when you choose to become an active validator again.  Your node, however, would not be listed as a nominable validator; thus, any nominators issuing new or revisions to existing nominations would be unable to select your node.
```

Author: It is probably just how I see things.  Nominators nominate the validator's bond.  The bond references the physical node via session keys.  Does that sound right or do you think it is too confusing?

Reviewer: As the target audience for this doc is validators, this should be okay!

> When you become an active validator you will also need to reset your validator preferences (commission, etc.). These can be configured as the same values that were set previously or something different.

Reviewer: ```suggestion
When you become an active validator, you will also need to reset your validator preferences (commission, etc.). These can be configured as the same values set previously or something different.
```

## Involuntary Chills

> If a validator was unresponsive for an entire session the validator bond would be chilled in a process known as _involuntary chilling._ When a validator has been involuntarily chilled, it may restrict the validator from being selected in the next election depending on the session in which it was chilled (see considerations above).  A validator that was chilled may re-declare the intent to validate at any time, however, it is recommended that the validator attempt to determine the source of the chill before doing so.

Reviewer: ```suggestion
If a validator was unresponsive for an entire session, the validator bond would be chilled in a process known as _involuntary chilling._ When a validator has been involuntarily chilled, it may restrict the validator from being selected in the next election depending on the session in which it was chilled (see considerations above).  A chilled validator may re-declare the intent to validate at any time. However, it is recommended that the validator attempts to determine the source of the chill before doing so.
```

> Slashing may also result in an involuntary chill, however, in this scenario the validator would also lose their nominations.  By this action, even if the validator re-declares his intent to validate before session 5, there were wouldn't be sufficient nominations to re-elect the bond into the active set.

Reviewer: ```suggestion
Slashing may also result in an involuntary chill. However, in that scenario, the validator would also lose their nominations.  By this action, even if the validator re-declares its intent to validate before session 5, there wouldn't be sufficient nominations to re-elect the node into the active set.
```

> Nominators who have the option to renominate a slashed validator using a display row in Polkadot-JS Apps. This row is displayed in the "Account Actions" tab for the nominator under a heading that says "Renomination required". 

Reviewer: ```suggestion
Nominators have the option to renominate a slashed validator using a display row in Polkadot-JS UI. This row is displayed in the "Account Actions" tab for the nominator under a heading that says "Renomination required". 
```
